packages/palette/src/elements/AutocompleteInput/AutocompleteInput.tsx

```diff
@@ -97,6 +99,7 @@ export const AutocompleteInput = <T extends AutocompleteInputOptionType>({
   dropdownMaxHeight = 308, // 308 = roughly 5.5 options
   flip = true,
   footer,
+  forwardRef: forwardedRef,
```

Comment:
Kind of the easiest way to deal workaround limitations with `forwardRef` and generics. Just pass it as a prop! Easy!